stripe/src/main/java/com/stripe/android/payments/StripeBrowserLauncherActivity.kt

```diff
@@ -56,9 +57,20 @@ internal class StripeBrowserLauncherActivity : AppCompatActivity() {
         viewModel.logCapabilities(shouldUseCustomTabs)
 
         if (shouldUseCustomTabs) {
+            val customTabColorSchemeParams = args.statusBarColor?.let { statusBarColor ->
+                CustomTabColorSchemeParams.Builder()
+                    .setToolbarColor(statusBarColor)
+                    .build()
+            }
+
             // use Custom Tabs
             val customTabsIntent = CustomTabsIntent.Builder()
                 .setShareState(CustomTabsIntent.SHARE_STATE_OFF)
+                .also {
+                    if (customTabColorSchemeParams != null) {
```

Comment:
`customTabColorSchemeParams?.let`

Reply:
Unfortunately `setDefaultColorSchemeParams()` requires a non-null argument, so I need to add the null check.

Reply:
hmm isn't `?.let` rules out the case when it's null? I tried the follow it seems to work
```
customTabColorSchemeParams?.let { params->
    it.setDefaultColorSchemeParams(params)
}
```

Reply:
Hm I think this is fine?